docs/data/rpc/api-reference/methods/sendTransaction.mdx

```diff
@@ -4,5 +4,143 @@ description: Submits a transaction
 ---
 
 import { RpcMethod } from "@site/src/components/RpcMethod";
+import { CodeExample } from "@site/src/components/CodeExample";
 
 <RpcMethod method="sendTransaction" platform="soroban" />
+
+### SDK Guide
+
+The example above is sending a transaction using RPC methods directly. If you are using the Stellar SDK to build applications, you can use the native functions to get the same information.
+
+<div>
+<CodeExample>
+
+```python
+# pip install --upgrade stellar-sdk
+from stellar_sdk import SorobanServer, soroban_rpc, Account, Asset, Keypair, Network, TransactionBuilder
+
+def send_transaction() -> soroban_rpc.SendTransactionResponse:
+    server = SorobanServer(server_url='https://soroban-testnet.stellar.org', client=None)
+
+    root_keypair = Keypair.from_secret(
+        "SA6XHAH4GNLRWWWF6TEVEWNS44CBNFAJWHWOPZCVZOUXSQA7BOYN7XHC"
+    )
+    # Create an Account object from an address and sequence number.
+    root_account = Account(account=root_keypair.public_key, sequence=1)
+
+    transaction = (
+        TransactionBuilder(
+            source_account=root_account,
+            # If you want to submit to pubnet, you need to change `network_passphrase` to `Network.PUBLIC_NETWORK_PASSPHRASE`
+            network_passphrase=Network.TESTNET_NETWORK_PASSPHRASE,
+            base_fee=100,
+        )
+        .append_payment_op(  # add a payment operation to the transaction
+            destination="GASOCNHNNLYFNMDJYQ3XFMI7BYHIOCFW3GJEOWRPEGK2TDPGTG2E5EDW",
+            asset=Asset.native(),
+            amount="125.5",
+        )
+        .append_set_options_op(  # add a set options operation to the transaction
+            home_domain="overcat.me"
+        )
+        .set_timeout(30)
+        .build()
+    )  # mark this transaction as valid only for the next 30 seconds
```

Comment:
This comment is probably more useful and informative if you move it up right above the line with `.set_timeout(30)`